tests/src/org.ehcache/ehcache/3.10.8-jakarta/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+
+import org.graalvm.internal.tck.TestUtils
+
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = TestUtils.testedLibraryVersion.split("-")[0]
+
+dependencies {
+    testImplementation("org.ehcache:ehcache:$libraryVersion") {
+        capabilities {
+            requireCapability('org.ehcache:ehcache-jakarta')
+        }
+    }
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+    testImplementation 'javax.cache:cache-api:1.1.1'
+    testImplementation 'org.slf4j:slf4j-api:2.0.4'
+    testImplementation 'ch.qos.logback:logback-classic:1.4.5'
+    testImplementation 'ch.qos.logback:logback-core:1.4.5'
+}
+
```

Comment:
- Now execute `./gradlew clean test -Pcoordinates=org.ehcache:ehcache:3.10.8-jakarta`, we will see a Warning Log. Do i need to do something or ignore it?
```shell
04:12:09.769 [main] WARN org.ehcache.sizeof.ObjectGraphWalker - The JVM is preventing Ehcache from accessing the subgraph beneath 'private final byte[] java.lang.String.value' - cache sizes may be underestimated as a result
java.lang.reflect.InaccessibleObjectException: Unable to make field private final byte[] java.lang.String.value accessible: module java.base does not "opens java.lang" to unnamed module @3e7dfd44
        at java.base@17.0.5/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base@17.0.5/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base@17.0.5/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
        at java.base@17.0.5/java.lang.reflect.Field.setAccessible(Field.java:172)
        at org.ehcache.sizeof.ObjectGraphWalker.getAllFields(ObjectGraphWalker.java:245)
        at org.ehcache.sizeof.ObjectGraphWalker.getFilteredFields(ObjectGraphWalker.java:204)
        at org.ehcache.sizeof.ObjectGraphWalker.walk(ObjectGraphWalker.java:159)
        at org.ehcache.sizeof.SizeOf.deepSizeOf(SizeOf.java:70)
        at org.ehcache.impl.internal.sizeof.DefaultSizeOfEngine.sizeof(DefaultSizeOfEngine.java:51)
        at org.ehcache.impl.internal.store.heap.OnHeapStore.getSizeOfKeyValuePairs(OnHeapStore.java:981)
        at org.ehcache.impl.internal.store.heap.OnHeapStore.makeValue(OnHeapStore.java:1529)
        at org.ehcache.impl.internal.store.heap.OnHeapStore.makeValue(OnHeapStore.java:1515)
        at org.ehcache.impl.internal.store.heap.OnHeapStore.newCreateValueHolder(OnHeapStore.java:1464)
        at org.ehcache.impl.internal.store.heap.OnHeapStore.lambda$put$8(OnHeapStore.java:344)
        at org.ehcache.impl.internal.concurrent.ConcurrentHashMap.compute(ConcurrentHashMap.java:1934)
        at org.ehcache.impl.internal.store.heap.SimpleBackend.compute(SimpleBackend.java:98)
```